src/aat/java/uk/gov/hmcts/reform/em/annotation/testutil/TestUtil.java

```diff
@@ -2,36 +2,56 @@
 
 import io.restassured.RestAssured;
 import io.restassured.specification.RequestSpecification;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.em.test.idam.IdamHelper;
+import uk.gov.hmcts.reform.em.test.s2s.S2sHelper;
 
-public class TestUtil {
-
-    private final String idamAuth;
-    private final String s2sAuth;
+import javax.annotation.PostConstruct;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
-    public TestUtil() {
-        IdamHelper idamHelper = new IdamHelper(
-            Env.getIdamUrl(),
-            Env.getOAuthClient(),
-            Env.getOAuthSecret(),
-            Env.getOAuthRedirect()
-        );
-
-        S2sHelper s2sHelper = new S2sHelper(
-            Env.getS2sUrl(),
-            Env.getS2sSecret(),
-            Env.getS2sMicroservice()
-        );
+@Service
+public class TestUtil {
 
+    @Autowired
+    private IdamHelper idamHelper;
+
+    @Autowired
+    private S2sHelper s2sHelper;
+
+    @Value("${test.url}")
+    private String testUrl;
+    @Value("${idam.api.url}")
+    private String idamAPIUrl;
+    @Value("${idam.client.id}")
+    private String idamClientId;
+    @Value("${idam.client.secret}")
+    private String idamClientSecret;
+    @Value("${idam.client.redirect_uri}")
+    private String idamClientRedirectUrl;
+
+    @PostConstruct
+    void postConstruct() {
+        printAllEnv();
         RestAssured.useRelaxedHTTPSValidation();
-
-        idamAuth = idamHelper.getIdamToken();
-        s2sAuth = s2sHelper.getS2sToken();
+        idamHelper.createUser("a@b.com", Stream.of("caseworker").collect(Collectors.toList()));
     }
 
     public RequestSpecification authRequest() {
         return RestAssured
             .given()
-            .header("Authorization", idamAuth)
-            .header("ServiceAuthorization", s2sAuth);
+            .header("Authorization", idamHelper.authenticateUser("a@b.com"))
+            .header("ServiceAuthorization", s2sHelper.getS2sToken());
+    }
+
+    private void printAllEnv() {
+        System.out.println(String.format("SYS_VARS"));
+        System.out.println(String.format("testUrl=%s", testUrl));
+        System.out.println(String.format("idamAPIUrl=%s", idamAPIUrl));
+        System.out.println(String.format("idamClientId=%s", idamClientId));
+        System.out.println(String.format("idamClientSecret=%s", idamClientSecret));
+        System.out.println(String.format("idamClientRedirectUrl=%s", idamClientRedirectUrl));
```

Comment:
Maybe use the logger?

Reply:
I'll remove it - it's was just debugging.